runtime/src/error.c

```diff
@@ -79,6 +80,47 @@ static void spinhaltIfAlreadyExiting(void) {
 #endif
 }
 
+// TODO This define should be set if CHPL_UNWIND= libunwind
+#define CHPL_UNWIND_LIBUNWIND
+
+#ifdef CHPL_UNWIND_LIBUNWIND
+#include<libunwind.h>
+// Necessary for instruct libunwind to use only the local unwind
+#define UNW_LOCAL_ONLY
+
+static void chpl_stack_unwind(){
+// This is just a prototype using libunwind
+  unw_cursor_t cursor;
+  unw_context_t uc;
+  unw_word_t wordValue;
+  char buffer[32];
+
+  unw_getcontext(&uc);
+  unw_init_local(&cursor, &uc);
+
+  if(chpl_sizeSymTable > 0)
+    fprintf(stderr,"\nStacktrace\n\n");
+
+  while (unw_step(&cursor) > 0) {                       // Stack unwind
+    // Currently it only takes the first 31 character
+    // of the symbol. Often this is more than enough.
+    unw_get_proc_name(&cursor, buffer, 32, &wordValue); // Native symbol
+    for(int t = 0; t < chpl_sizeSymTable; t+=2 ){       // Explore symbol table
```

Comment:
Could you add a comment here:

Since this stack trace is printed out a program exit, we do not believe it is performance sensitive. Additionally, this initial implementation favors simplicity over performance.
If it becomes necessary to improve performance, this code could use  be faster using one of these two strategies:
 1) Use a hashtable or map to find entries in chpl_funSymTable, or
 2) Emit chpl_funSymTable in sorted order and use binary search on it